packages/rspack-cli/src/rspack-cli.ts

```diff
@@ -163,10 +163,6 @@ export class RspackCLI {
 				item.mode = options.mode as Mode;
 			}
 
-			// false is also a valid value for sourcemap, so don't override it
-			if (typeof item.devtool === "undefined") {
-				item.devtool = isBuild ? "source-map" : "cheap-module-source-map";
```

Comment:
this will not executed when devtool is set to false

Reply:
but it cause the default value of devtool is `source-map` or `cheap-module-source-map` instead of `false`

Reply:
and this makes it not working https://github.com/web-infra-dev/rspack/blob/40fef768bfe8d9c495f6a6ea5c7a13c073517ecd/packages/rspack/src/config/defaults.ts#L66